src/events/Windows.cpp

```diff
@@ -321,6 +321,7 @@ void Events::listener_mapWindow(void* owner, void* data) {
                 try {
                     PWINDOW->m_closeableSince = Time::steadyNow() + std::chrono::milliseconds(std::stoull(VARS[1]));
                 } catch (std::exception& e) { Debug::log(ERR, "Rule \"{}\" failed with: {}", r->m_rule, e.what()); }
+                break;
```

Comment:
Added for consistency with other cases. I'll remove it if the lack of it was intended.